web/src/app/users/UserContactMethodList.js

```diff
@@ -34,6 +37,9 @@ const testCM = gql`
   }
 `
 
+const DISABLED_CM_TOOLTIP =
+  'Contact method disabled. See its options to reactivate.'
```

Comment:
For sure -- I wasn't able to find a sentence that felt good, so I went with this for the time being. Since it's a tooltip we will want to keep it as concise as possible, since this is an overlay on the rest of the page.

If we can't think of good wording for the tooltip in just a few words perhaps we can discuss showing the warning in-line under or near the warning icon.

Reply:
On second thought I think we should approach this differently after chatting with @dayander for a bit. A tooltip might not even be the way to go, as it basically forces the user to first find hover over the tooltip to get the information, click away, find the options menu, click it, then open the verification dialog from that menu. That's a total of **3 clicks, and a hover** (over the tooltip) to get to the dialog. Maybe we can get the team together and sketch a few designs out (like we did with Sylas) and go from there. @m17ch @arurao @mastercactapus @Katy-M

Reply:
Yeh - that sounds good @Auchindoun -- I did think clicking "Verify / Reactivate" from menu on CM then clicking on "Send" also felt a bit unnecessary.

I think having a wholistic design/flow discussion would be very worthwhile!